client/src/scss/styles.scss

```diff
@@ -1 +1,42 @@
-@import "~bootstrap/scss/bootstrap.scss"
\ No newline at end of file
+@use 'variables' as v;
+/* Override preset styles */
+
+// Colors
+$primary: green;
+$secondary: v.$secondary-500;
+
+// Link Colors
+$link-color: $secondary;
+
+// Input & Select Colors
+$input-bg: v.$neutral-500;
+$input-border-color: v.$neutral-500;
+
+// Nav Dropdown
+.custom-dropdown {
+  .dropdown-item.active {
+     background-color: v.$neutral-300 !important;
+  }
+}
+
+// Gradient Effect on Text
+.gradient-text {
+  background-color: v.$neutral-50;
+  background-image: linear-gradient(0deg, #669F28, #B3EE74 );
+  background-size: 100%;
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent; 
+  -moz-text-fill-color: transparent;
+}
+
+// Large Text that isn't a Heading Tag
+.large-text {
+  font-size: calc(1.375rem + 1.5vw);
+  font-weight: 700;
+}
```

Comment:
styles.scss file is only for changing presets. The styles .large-text, .gradient-text, should be in the Header's style file.

'Font-weight: 700' already has a preset className 'font-weight-bold'

If 'font-size: calc(1.375rem + 1.5vw)' will be used often then it should be a variable in the variables.scss file

If the nav dropdown color is supposed to be consistent then you should change the preset in styles.scss by adding '$dropdown-link-active-bg: v.$neutral-300'

Reply:
Gotcha. I will take another looksy then. But if we wanted to use the gradient-text somewhere else in the site, shouldn't it stay in the styles.scss file? 
Thanks for the comments!

Reply:
put it after the import for now since everything above is supposed to be for bootstrap presets.